todo_app/app.py

```diff
@@ -11,132 +11,184 @@
 import pymongo
 from bson.objectid import ObjectId
 from datetime import datetime
+from flask_login import LoginManager
+from oauthlib.oauth2 import WebApplicationClient
+from flask_login import login_required
+from urllib import parse
+from flask import request
+from flask_login import UserMixin, login_user, current_user
+from todo_app.data.UserClass import User
+import json
+import logging
+from loggly.handlers import HTTPSHandler
+from logging import Formatter
+from logging import getLogger
 
 
 def create_app():
     app = Flask(__name__)
-
-    toDoId = os.environ['TOID']
-    doingId = os.environ['DOINGID']
-    doneId = os.environ['DONE']
-    dbconnect = os.environ['CLIENT']
+    githubId = os.environ['GITHUBID']
+    webClient = os.environ['WEBAPPLICATIONCLIENT']
+    clientSecret = os.environ['CLIENTSECRET']
+    todoBoard = os.environ['TODOBOARD']
+    doingBoard = os.environ['DOINGBOARD']
+    doneBoard = os.environ['DONEBOARD']
+    connectString = os.environ['CONNECTIONSTRING']
+    app.secret_key = os.getenv('SECRET_KEY')
+    app.config['LOGIN_DISABLED'] = os.getenv('LOGIN_DISABLED') == 'True'
+    app.logger.setLevel(os.getenv('LOG_LEVEL', 'INFO'))
+    # app.logger.setLevel(os.getenv('LOGGLY_TOKEN'))
+    if os.getenv('LOGGLY_TOKEN'):
+        handler = HTTPSHandler(
+            f'https://logs-01.loggly.com/inputs/{os.getenv("LOGGLY_TOKEN")}/tag/todo-app')
+        handler.setFormatter(
+            Formatter("[%(asctime)s] %(levelname)s in %(module)s: %(message)s"))
+        app.logger.addHandler(handler)
+    getLogger('werkzeug').addHandler(HTTPSHandler(
+        f'https://logs-01.loggly.com/inputs/{os.getenv("LOGGLY_TOKEN")}/tag/todoapp-requests'))
```

Comment:
Should be in `if` block
```suggestion
        getLogger('werkzeug').addHandler(HTTPSHandler(
            f'https://logs-01.loggly.com/inputs/{os.getenv("LOGGLY_TOKEN")}/tag/todoapp-requests'))
```